doc_source/security_iam_resource-based-policy-examples.md

```diff
@@ -41,7 +41,11 @@ To access the AWS Elemental MediaConnect console, you must have a policy that de
                 "iam:PassRole"
             ],
             "Effect": "Allow",
-            "Resource": "*"
+            "Resource": "*",
+            "Condition": {
+                "StringLike": {
+                    "iam:PassedToService": "mediaconnect.amazonaws.com"
```

Comment:
This block is missing the closed curly brackets

Reply:
Done. Validated the json this time. Sorry about that miss!